### DISCUSSION TITLE
UI for Fine Tuning

### DISCUSSION THREAD

**Discussion by wakaztahir** [LAION-AI/Open-Assistant#3405] [Ideas]
How can I fine tune it on my own data

OpenAI has some thirdparty UI's that can do fine tuning easily using data from spreadsheets , inputs and outputs , It would be really helpful if UI like this could be provided and fine tuning could be made easier for people who don't want to mess with code

**Comment by jimafisk** [comment]
Related ticket: https://github.com/LAION-AI/Open-Assistant/issues/2341

Not sure if xTuring (https://github.com/stochasticai/xturing) could be extended to help with this?

**Comment by ThomasBourgeois** [comment]
Interested, even if there needs to be some coding involved too.
